### PR TITLE
adamCzyzBranchIdea

### PR DESCRIPTION
**Pull request by niebieskiczlowiek**
Created an endpoint under /superFunEndpoint with a message coded in base64

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+from starlette.responses import PlainTextResponse
 from starlette.routing import Route
+import base64
 
 
 async def homepage(request):
     return JSONResponse({'hello': 'world'})
 
 
+async def superFunEndpoint(request):
```
Comment (t1waz) [inline backend/main.py]: method names in python are always snake_case not CamelCase.

pls refactor

File: backend/main.py [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+    """))
+
+
 app = Starlette(debug=True, routes=[
     Route('/hello', homepage),
+    Route('/superFunEndpoint', superFunEndpoint),
```
Comment (t1waz) [inline backend/main.py]: consider changing name to capybara

**Review by t1waz — CHANGES_REQUESTED** [review]
pls refactor CamelCase to snake_case

**Comment by t1waz** [comment]
pls tell rest of students that there is deadline with PR until next meet :)

**Comment by niebieskiczlowiek** [comment]
refactored method name to capybara_endpoint and the endpoint name to just capybara (and reminded students of deadline :) )

**Review by t1waz — APPROVED** [review]
LGTM

**Comment by t1waz** [comment]
pls rebase

**Comment by t1waz** [comment]
pls send this discord to other students (we will try to use it for communication) https://discord.gg/fpuqSQBk

**Pull request merged**